nfc/libs/NfcCoreLib/lib/Nci/phNciNfc_Init.c

```diff
@@ -69,16 +100,164 @@ static NFCSTATUS phNciNfc_Init(void *pContext)
 static NFCSTATUS phNciNfc_ProcessInitRsp(void *pContext, NFCSTATUS Status)
 {
     NFCSTATUS wStatus = Status;
+    pphNciNfc_Context_t pNciContext = pContext;
+
+    PH_LOG_NCI_FUNC_ENTRY();
+    if (NULL != pNciContext)
+    {
+        if (PH_NCINFC_VERSION_IS_1x(pNciContext))
+        {
+            wStatus = phNciNfc_ProcessInitRspNci1x(pContext, Status);
+        }
+        else if (PH_NCINFC_VERSION_IS_2x(pNciContext))
+        {
+            wStatus = phNciNfc_ProcessInitRspNci2x(pContext, Status);
+        }
```

Comment:
ok. I agree with this. I will return NFCSTATUS_INVALID_PARAMETER as a failure code in case we are not NCI1x nor NCI2x.